packages/perseus/src/components/sortable.tsx

```diff
@@ -582,8 +586,7 @@ class Sortable extends React.Component<SortableProps, SortableState> {
         });
 
         this.setState({items}, () => {
-            this.props.onMeasure &&
-                this.props.onMeasure({widths: widths, heights: heights});
```

Comment:
This is another case that `@typescript-eslint/no-unnecessary-condition` picked up.